docs/CONTRIBUTING.md

```diff
@@ -98,8 +98,9 @@ View Components are either functional and written in React, or non-functional an
 
 ### React components
 
-- The JS for each component lives in `app/javascript/components/**/XXX.js`
-- The CSS for each component lives in `app/css/components/**/XXX.css`
+- The JS for each component lives in `app/javascript/components/**/XXX.js`.
+- The CSS for each component lives in `app/css/components/**/XXX.css`.
```

Comment:
Is this a requirement for tailwind? Personally, I would suggest having the CSS for the component in the same directory as the component itself, as they're so tightly bound together.

Reply:
No, I just don't know how to do that.

If you can work out how to do that, I'd be :+1: on that too. I'll merge this and you can update it if you can fix it 🙂

Reply:
Shall I create an issue?

Reply:
Please :)